system/include/emscripten/val.h

```diff
@@ -190,6 +192,8 @@ namespace emscripten {
                 const void* p;
             } w[2];
             double d;
+            int64_t i;
+            uint64_t u;
```

Comment:
do we need both signed and unsigned? i'd hope only unsigned is enough, as with `unsigned u` above for 32-bit values.

Reply:
@kripken You are correct, the `int64_t` member was redundant.